build.gradle

```diff
@@ -24,22 +24,22 @@ allprojects {
 }
 
 subprojects {
-    if(it.parent.name == 'examples') {
+    if(it.parent.name == 'examples' || it.parent.name == 'test') {
         apply plugin: 'java'
     } else {
         apply plugin: 'java-library'
         apply plugin: 'jacoco'
 
         dependencies {
             // Testing
-            testImplementation "org.junit.jupiter:junit-jupiter-api:$junitJupiterVersion"
-            testRuntimeOnly "org.junit.jupiter:junit-jupiter-engine:$junitJupiterVersion"
-            testImplementation "org.mockito:mockito-core:$mockitoVersion"
-            testImplementation "org.mockito:mockito-junit-jupiter:$mockitoVersion"
+            testImplementation(libs.junit.jupiter.api)
+            testImplementation(libs.junit.jupiter.engine)
+            testImplementation(libs.mockito.core)
+            testImplementation(libs.mockito.junit.jupiter)
         }
 
         jacoco {
-            toolVersion = jacocoVersion
+            toolVersion = "0.8.7"
```

Comment:
Can't we use something like [this](https://docs.gradle.org/current/userguide/platforms.html#sec:common-version-numbers)?
```
checkstyle {
    // will use the version declared in the catalog
    toolVersion = libs.versions.checkstyle.get()
}
```

Reply:
good point 👍 done ✅